### PR TITLE
fix(lsp): ensure `enablePaths` works when clients do not provide a trailing slash for workspace dir

### PR DESCRIPTION
**Pull request by dsherret**
Closes https://github.com/denoland/vscode_deno/issues/827

### CODE REVIEW

File: cli/lsp/language_server.rs [hunk 1]
```diff
@@ -377,7 +377,7 @@ impl LanguageServer {
       let mut ls = self.0.write().await;
       if let Ok(configs) = configs_result {
         for (value, internal_uri) in
-          configs.into_iter().zip(specifiers.into_iter().map(|s| s.1))
+          configs.into_iter().zip(specifiers.into_iter().map(|s| s.0))
```
Comment (dsherret) [inline cli/lsp/language_server.rs:380]: I opened #18374.

Comment (bartlomieju) [inline cli/lsp/language_server.rs:380]: Maybe `specifiers` should be a struct with descriptive field names instead of a tuple?

**Review by bartlomieju — APPROVED** [review]
LGTM, nice fix!

File: cli/lsp/language_server.rs (hunk 1 above)
Comment (dsherret) [reply cli/lsp/language_server.rs:380]: Yes, I opened https://github.com/denoland/deno/issues/18374 for the future

**Pull request merged**